queries.yaml

```diff
@@ -929,8 +931,9 @@ post_process:
         ] if area >= area_threshold][0]
       # note the following line cannot be written as
       # properties is not None and properties.get('label_placement') is True
+      # and we exclude sea and ocean because those are pre-graded
       where: >-
-        properties is not None and label_placement
+        properties is not None and label_placement and kind not in ('sea', 'ocean')
```

Comment:
This exclusion is required else the OSM points for oceans and seas will get re-graded to min_zoom 16!